mdoc/Test/en.expected.typeForwards/TheNamespace/TheClass.xml

```diff
@@ -12,6 +12,17 @@
   <AssemblyInfo>
     <AssemblyName>DocTest-typeForwards-Second-First</AssemblyName>
   </AssemblyInfo>
+  <AssemblyInfo>
+    <AssemblyName>DocTest-typeForwards-Third</AssemblyName>
+    <AssemblyVersion>0.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <AssemblyInfo>
+    <AssemblyName>DocTest-typeForwards-Third-First</AssemblyName>
```

Comment:
This isn't strictly in the scope of this particular task, but since you're already going to look at it again for the other comment ... can you find out why this doesn't have an `AssemblyVersion` ? Making sure that we properly log all the AssemblyVersion information would go a long way to helping us figure out resolution issues when they occur :)

Reply:
Good catch, also noticed `AssemblyVersion` missing both for type and members of the "from" assembly(such as DocTest-typeForwards-Second-First).

Did some investigation, this could related to the lines here

[For Type](https://github.com/mono/api-doc-tools/blob/develop/mdoc/Mono.Documentation/MDocUpdater.cs#L2219)
`
Func<XmlElement, bool> assemblyFilter = x => x.SelectSingleNode ("AssemblyName").InnerText == type.Module.Assembly.Name.Name;
`

Take DocTest-typeForwards-Second-First.dll for example, the type.module and member.module will return "DocTest-typeForwards-Second.dll".

So the node `<AssemblyName>DocTest-typeForwards-Second-First</AssemblyName>` will never be picked up to update assembly version.

I think we may need another bug to track the issue and discuss the fix of it.

![type-module](https://user-images.githubusercontent.com/33647870/72508489-b3b25680-3880-11ea-9066-61b4182484d3.PNG)

Reply:
There *is* a GH bug which discusses this very issue (#399) ... but no internal item, so I've [created one](https://ceapex.visualstudio.com/Engineering/_workitems/edit/163956) that we can address separately :) /cc @mimisasouvanh